crates/ironclaw_gateway/static/styles/surfaces/memory.css

```
@@ -1,343 +1,355 @@
-  border: 1px solid var(--border);
-  border-radius: var(--radius);
-  color: var(--text);
+  overflow-y: auto;
+  padding: var(--v2-space-6) var(--v2-space-8);
   font-family: var(--font-mono);
   font-size: var(--text-sm);
-  line-height: 1.5;
-  resize: none;
+  line-height: 1.6;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)